docs/_documentations/troubleshooting.md

```diff
@@ -1002,6 +1002,20 @@ If you can run `docker` from the command line, but Codewind still fails to find
 3. Open the terminal you ran `docker` from and run `code`.
 4. Now, the new VS Code instance shares the terminal's `PATH`. Start Codewind again.
 
+## VS Code extension fails to start on Ubuntu
+<!--
+Action/Topic: VS Code extension fails to start on Ubuntu
```

Comment:
Part 2 mentions MacOS, should this be shortened to "VSCode extension fails to start"

Reply:
Good catch, corrected throughout.